src/resolve/resolved-def.h

```diff
@@ -84,3 +84,7 @@
 #define SD_RESOLVED_FROM_MASK       (SD_RESOLVED_FROM_CACHE|SD_RESOLVED_FROM_ZONE|SD_RESOLVED_FROM_TRUST_ANCHOR|SD_RESOLVED_FROM_NETWORK)
 
 #define SD_RESOLVED_QUERY_TIMEOUT_USEC (120 * USEC_PER_SEC)
+
+/* Should be longer than transaction timeout for a single UDP transaction, so we get at least
+ * one transaction retry before timeouting the whole candidate */
+#define SD_RESOLVED_CANDIDATE_TIMEOUT_USEC (6 * USEC_PER_SEC)
```

Comment:
Maybe this should be rewritten as `SD_RESOLVED_TRANSACTION_TIMEOUT + 1*USEC_PER_SEC`? That would require moving some defines around in headers.

Reply:
yeah, i think that would make sense.

Reply:
maybe call this SD_RESOLVED_QUERY_EXPEDITED_TIMEOUT_USEC?

or alternatively some name that includes "partial success" or so?

Reply:
I've done this now. Ended up moving `DNS_TRANSACTION_ATTEMPTS_MAX` too, since that is used to define the UDP transaction timeout. I figured it's better to also move the related defines (MDNS, LLMNR) for consistency, but ofc it makes the diff larger and git blame harder, so let me know if you'd rather have me not move everything around.